### ISSUE TITLE
MacOS: Not starting

### ISSUE THREAD

**Issue by ClaudiusCluever** [OpenRA/OpenRA#16973]
First of all, thank you for the good work!

I am on MacOS Mojave 10.14.06 and try to start OpenRA-release-20190314.

It doesn't even start but the OS presents me with an error message, saying that the file is damaged/corrupted and it is recommended to put it in the trash bin. (I am using the OS in german, so original error message below.) 

The current playtest does work. Any chance to get the release version to work?

Original Error message:
"„OpenRA - Red Alert“ ist beschädigt und kann nicht geöffnet werden. Es empfiehlt sich, das Objekt in den Papierkorb zu bewegen."

**Comment by pchote** [comment]
Try following the instructions in this screenshot:

![](https://user-images.githubusercontent.com/167819/63227878-63589f00-c1e3-11e9-8873-1b9a43b32e1d.png)

Does this help?

**Comment by ClaudiusCluever** [comment]
No, i know that issue. I just checked again, just to be sure. "Open Anyway" doesn't appear in the system preferences in this case.

The error message you are referring to doesn't mention anything about OpenRA being damaged/corrupted/broken ("beschädigt").

**Comment by pchote** [comment]
In that case I suspect that something really is damaged/corrupted on your side - if not the OpenRA .app then probably your system Mono installation. The pre-playtest builds on the forum include their own runtime, so wouldn't be affected by a corrupted Mono install.

Try downloading and reinstalling Mono and OpenRA. The release-20190314 build from the website runs without issue on my 10.14.6 install.

**Comment by ClaudiusCluever** [comment]
Did that, still the same. Is it possible that Mono is broken even after uninstall and reinstall?

**Comment by pchote** [comment]
Try opening a terminal and launching directly from the commandline.

For example, on my system I see:
```
paul@Ragnarok:~$ /Applications/OpenRA\ -\ Red\ Alert.app/Contents/MacOS/OpenRA
2019-08-22 14:36:01.933 OpenRA[19200:2648863] Running launchgame with arguments:
2019-08-22 14:36:01.933 OpenRA[19200:2648863] --debug
2019-08-22 14:36:01.933 OpenRA[19200:2648863] /Applications/OpenRA - Red Alert.app/Contents/Resources/OpenRA.Game.exe
2019-08-22 14:36:01.933 OpenRA[19200:2648863] Engine.LaunchPath="/Applications/OpenRA - Red Alert.app/Contents/MacOS/OpenRA"
2019-08-22 14:36:01.934 OpenRA[19200:2648863] Game.Mod=ra
2019-08-22 14:36:01.934 OpenRA[19200:2648863] /Applications/OpenRA - Red Alert.app/Contents/MacOS/OpenRA
Platform is OSX
Engine version is release-20190314
Using SDL 2 with OpenGL renderer
... <rest of normal OpenRA terminal output removed> ...
```

For you I expect it should print some macOS debug messages explaining why it thinks it is damaged.

**Comment by ClaudiusCluever** [comment]
Okay, that worked! It started. This is what the terminal recorded:

`Platform is OSX
Engine version is release-20190314
Using SDL 2 with OpenGL renderer
Desktop resolution: 2560x1440
No custom resolution provided, using desktop resolution
Using resolution: 2560x1440
2019-08-22 15:59:07.255 launchgame[3468:85610] ApplePersistenceIgnoreState: Existing state will not be touched. New state will be written to (null)
Using window scale 1.00
OpenGL version: 2.1 NVIDIA-12.0.24 355.11.10.50.10.103
Using default sound device
Internal mods:
	ra: Red Alert (release-20190314)
	modcontent: Mod Content Manager (release-20190314)
External mods:
	cnc-release-20181215: Tiberian Dawn (release-20181215)
	ra-release-20190314: Red Alert (release-20190314)
Loading mod: ra
2019-08-22 15:59:15.769 OpenRA[3467:85568] launchgame exited with code 0
`

**Comment by PunkPun** [comment]
Is this still an issue on the current release? If yes is it a problem on the devtest? https://forum.openra.net/viewtopic.php?f=82&t=21649

**Comment by PunkPun** [comment]
Closing as outdated
